cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -809,5 +812,6 @@ func init() {
 	createCmd.Flags().StringVar(&configPatch, "config-patch", "", "patch generated machineconfigs (applied to all node types)")
 	createCmd.Flags().StringVar(&configPatchControlPlane, "config-patch-control-plane", "", "patch generated machineconfigs (applied to 'init' and 'controlplane' types)")
 	createCmd.Flags().StringVar(&configPatchJoin, "config-patch-join", "", "patch generated machineconfigs (applied to 'join' type)")
+	createCmd.Flags().BoolVar(&badRTC, "bad-rtc", false, "launch VM with bad RTC state (QEMU only)")
```

Comment:
would be nice to put such test flags under some separate sandbox like:

```
-test-flags bad-rtc...
```

Reply:
yeah, agree, I would hide 80% of the flags somewhere :)

like `--expert-mode` )